README.md

```diff
@@ -91,4 +116,4 @@ Print job POST request bodies will look something like this:
 }
 ```
 
-See documentation on port `9292`, endpoint `/v1`.
+For more details, see the built-in API 1 documentation at http://localhost:9292/v1
```

Comment:
I just want to point out that `PrintersController` and `LabelTemplatesController` in v1 and v2 are exactly the same. If we had controllers for v2 and the routes, we could migrate v1 docs to v2 docs in the same format. It may be for another story. No action necessary for this PR.

Reply:
v1 is toshiba only. 

v2:
- supports toshiba and squix and has a simplified the print job. It means the user / client can select a printer without needing to know what type it is and using a single print job. Underneath the hood squix and toshiba are very different. If the printer is Squix it will create a squix specific print job and forward it on.

A lot of the endpoints return the same thing in v1 and v2 but due to JSON API resource they need to have separate resources.